<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusReportPage</class>
 <widget class="QDialog" name="StatusReportPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Status Report</string>
  </property>
  <widget class="QLabel" name="bg_dark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #22223b;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg_light">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>1152</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4a4e69;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(242, 233, 228);
</string>
   </property>
   <property name="text">
    <string>Status Report</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="unpaidScrollArea">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>531</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(154, 140, 152);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>199</height>
     </rect>
    </property>
    <widget class="QScrollBar" name="unpaidSlider">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>0</y>
       <width>16</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar::handle {
	background-color: #4a4e69;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>600</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15;</string>
   </property>
   <property name="text">
    <string>Export report as CSV</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="unusedRooms">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>383</y>
     <width>531</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(154, 140, 152);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>199</height>
     </rect>
    </property>
    <widget class="QScrollBar" name="unusedRoomSlider">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>0</y>
       <width>16</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar::handle {
	background-color: #4a4e69;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>481</width>
     <height>207</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="totalCustomerLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Total Customers:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="paidCustomerLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string># of Paid Customers:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unpaidCustomerLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string># of Unpaid Customers:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="revenueLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Total Revenue:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>481</width>
     <height>207</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="areaUtilizedLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Area Utilized:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unusedAreaLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Unused Area:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unusedVolumeLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Unused Volume:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="capacityLabel">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Capacity Utilization:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
